### PR TITLE
WLS Life Cycle feature 

### PR DESCRIPTION
**Pull request by vikashsonu900**

### CODE REVIEW

File: ansible-fmw/wls_variables.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ JDK_DIR: '{{ JAVA_DIR }}/jdk1.8.0_331'
 JAVA_HOME: '{{ JAVA_DIR }}/java_latest'
 
 ## Oracle ENV Variables
-ORACLE_BASE: "/home/dojowls/wls_lite"
+ORACLE_BASE: "/home/{{ WLS_USER }}/fmw"
```
Comment (zeusbaba) [inline ansible-fmw/wls_variables.yml]: `fmw` is general name of all Fusion Middleware which covers many :)  
this playbook is _only_  about WLS , therefore `wls-server` or even `wls-lite` is better choice.

**Review by zeusbaba — CHANGES_REQUESTED** [review]

File: ansible-fmw/wls_variables.yml (hunk 1 above)
Comment (vikashsonu900) [reply ansible-fmw/wls_variables.yml]: The changes have been done. Please check

**Review by zeusbaba — APPROVED** [review]
well done work! :-)

**Comment by zeusbaba** [comment]
@vikashsonu900 plz squash & merge this PR

**Pull request merged**